pipeline.py

```diff
@@ -109,7 +119,7 @@
     
     # Add survey data to the messages
     print("Combining Datasets...")
-    data = CombineRawDatasets.combine_raw_datasets(user, messages_datasets, [])
+    data = CombineRawDatasets.combine_raw_datasets(user, [demographics], messages_datasets)
```

Comment:
Why are you passing in demographics before the messages?

Reply:
The demographics dataset is our "master dataset", the full set of contacts. A subset of these will have answered the survey.

Reply:
Ok, that's interesting. This is obviously a bit confusing because it means you have to pass the messages dataset to the surveys arg and the surveys dataset to the messages arg. Please add a comment explaining why this is happening, and file an issue against the project for renaming the arguments to CombineRawDatasets.combine_raw_datasets

Reply:
Comment added.